src/openzeppelin/tests/security/test_pausable.cairo

```diff
@@ -1,7 +1,21 @@
+use option::OptionTrait;
+use starknet::contract_address_const;
+use starknet::ContractAddress;
+use starknet::testing;
+
 use openzeppelin::security::pausable::Pausable;
-use openzeppelin::security::pausable::Pausable::PausableImpl;
 use openzeppelin::security::pausable::Pausable::InternalImpl;
+use openzeppelin::security::pausable::Pausable::PausableImpl;
+use openzeppelin::security::pausable::Pausable::Paused;
+use openzeppelin::security::pausable::Pausable::Unpaused;
```

Comment:
Is an interesting idea, but for that, we need to have all the events in the same scope, and avoid potential clashes (similar to selectors). We could merge the imports for this case to avoid to many lines, but we need to come with a pattern for this then (for imports I mean).